setup.py

```diff
@@ -11,8 +11,7 @@
 setup(
     name='social_bot',
     version='0.0.1',
-    install_requires=['gym', 'numpy',
-                      'matplotlib'],  # And any other dependencies foo needs
+    install_requires=['gin-config', 'gym', 'numpy', 'matplotlib'],
```

Comment:
The README has manual installation for these libs. Maybe change it to running setup.py? (also include 'psutil')

Reply:
Changed